src/decoder/symphonia.rs

```diff
@@ -72,13 +72,8 @@ impl SymphoniaDecoder {
             None => return Ok(None),
         };
 
-        let mut decoder = symphonia::default::get_codecs().make(
-            &stream.codec_params,
-            &DecoderOptions {
-                verify: true,
-                ..Default::default()
-            },
-        )?;
+        let mut decoder = symphonia::default::get_codecs()
+            .make(&stream.codec_params, &DecoderOptions { verify: true })?;
```

Comment:
`pub struct DecoderOptions {` only has `pub verify: bool,`, which this code already specifies, so `..Default::default()` isn't doing anything here.